config/default.yml

```diff
@@ -109,6 +109,22 @@ RSpec/EmptyLineAfterSubject:
   Enabled: true
   StyleGuide: http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/EmptyLineAfterSubject
 
+RSpec/ExampleGroupMethods:
+  Description: This cop enforces custom example group definitions methods.
+  Enabled: false
+  PreferredMethods:
+    describe: describe
```

Comment:
I guess there is no need to set a preference for ones that remain the same.
You can set some default, like using `describe` instead of `example_group` 
Such a cop would be useful for examples as well (using `it` instead of an `example`, for example.).

Reply:
I can extend it to catch all of the RSpec methods. Call it `RSpec/RSpecPreferredMethods` (still may need a better name) and use the `Language::ALL.block_pattern` that will allow all the group/examples/hooks/subject/etc definition methods. Then I can set the proposed defaults.

Reply:
@bquorning WDYT? One generic cop, separate cops per group?

Reply:
We already have several cops that perform a similar function of enforcing a particular alias. (i.e. [RSpec/ItBehavesLike](https://github.com/rubocop-hq/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/it_behaves_like.rb)). If we want to have a PreferedMethods / PreferedAliases / WhateverWeCallThisThing cop I wonder if we should (eventually?) consolidate all the existing cops that perform the same kind of function. This could also include more things like [RSpec/NotToNot](https://github.com/rubocop-hq/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/not_to_not.rb) if we wanted it to.

Reply:
There are a large number of RSpec aliases we haven't covered yet with existing cops and it's not always clear if they merit their own cops. For example, #727 mentions several.

Reply:
Decided to generalize it and extend it to all them RSpec methods. Called it `RSpec/Dialect` as you can choose your own dialect, in a way. Tell me what you folks think.

Reply:
I don't think Dialect is a bad name, necessarily, but my current suggestion would be `PreferredAliases` i think because i expect that would be more self-evident, though "dialect" is probably clear enough once you understand what it does. But without knowing what the cop does, i think `PreferredAliases` might be a hair more descriptive.

Not a blocker, just my $0.02.